### ISSUE TITLE
$VERSION is outside the package

### ISSUE THREAD

**Issue by barefootcoder** [Csson/p5-Dist-Zilla-Plugin-PodnameFromClassname#1]
Your `$VERSION` variable is not inside your actual `package` declaration.  This leads to the following error:

```
perl -e 'use Dist::Zilla::Plugin::PodnameFromClassname 0.1002'
Dist::Zilla::Plugin::PodnameFromClassname does not define $Dist::Zilla::Plugin::PodnameFromClassname::VERSION--version check failed at -e line 1.
BEGIN failed--compilation aborted at -e line 1.
```

The solution is just to move the declaration of `$VERSION` inside the package.  Thanx!

**Comment by Csson** [comment]
Thanks for catching that. It's fixed in 0.1100.

I also removed the rules regarding empty lines before and after the comment line. Not sure what the purpose of that was.

**Comment by barefootcoder** [comment]
> Thanks for catching that. It's fixed in 0.1100.

Awesome.

> I also removed the rules regarding empty lines before and after the comment line. Not sure what the purpose of that was.

Extra awesome! :-)

Thx for the quick turn-around.
